Content/Items/Gear/Weapons/Battleaxe/CorruptedBattleaxe.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace PathOfTerraria.Content.Items.Gear.Weapons.Battleaxe;
+
+internal class CorruptedBattleaxe : IronBattleaxe
+{
+	public override string Texture => $"{PathOfTerraria.ModName}/Assets/Items/Gear/Weapons/Battleaxe/{GetType().Name}";
```

Comment:
This seems to happen consistently with new types of Gear - is there a reason this Texture override is copy-pasted to the two existing child classes instead of being defined in the parent? This'd save boilerplate for no downside, and still allow overriding the texture if desired.

Reply:
I think it was left over from when we didn't get GetType, but now it makes sense!
Updated